### PR TITLE
refactor: updating linux pkr.hcl

### PR DESCRIPTION
**Pull request by burnsjared0415**
# 

## Summary of Pull Request
Updating files for linux OS to simplify variables and build files

## Type of Pull Request

- [ ] This is a bugfix. `type/bug`
- [ ] This is an enhancement or feature. `type/feature` or `type/enhancement`
- [ ] This is a documentation update. `type/docs`
- [x] This is a refactoring update. `type/refactor`
- [ ] This is a chore. `type/chore`
- [ ] This is something else.
      Please describe:

## Related to Existing Issues

Issue Number: N/A

## Test and Documentation Coverage

- [x] Tests have been completed.
- [ ] Documentation has been added or updated.

## Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: builds/linux/debian/12/linux-debian.pkrvars.hcl.example [hunk 1]
```diff
@@ -7,40 +7,17 @@
...
-vm_disk_thin_provisioned = true
-vm_network_card          = "vmxnet3"
 
 // Removable Media Settings
 iso_datastore_path       = "iso/linux/debian"
 iso_content_library_item = "debian-12.5.0-amd64-netinst"
 iso_file                 = "debian-12.5.0-amd64-netinst.iso"
 
-// Boot Settings
-vm_boot_order = "-"
```
Comment (tenthirtyam) [inline builds/linux/debian/12/linux-debian.pkrvars.hcl.example:41]: Disk you copy "-" over as the default. Slightly different for Debian here.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
One comment. :)

File: builds/linux/debian/12/linux-debian.pkrvars.hcl.example (hunk 1 above)
Comment (burnsjared0415) [reply builds/linux/debian/12/linux-debian.pkrvars.hcl.example:41]: double checked and looks like it is set correct

**Review by tenthirtyam — APPROVED** [review]
LGTM

**Pull request merged**